docs/setup/settings.asciidoc

```diff
@@ -16,6 +16,8 @@ you'll need to update your `kibana.yml` file. You can also enable SSL and set a
 `elasticsearch.customHeaders:`:: *Default: `{}`* Header names and values to send to Elasticsearch. Any custom headers
 cannot be overwritten by client-side headers, regardless of the `elasticsearch.requestHeadersWhitelist` configuration.
 
+`elasticsearch.logQuerie:`:: *Default: `false`* When you turn on verbose logging and set `elasticsearch.logQueries: true` the Kibana Elasticsearch server client will log queries generated by the Kibana server. This is useful for seeing the query DSL generated by applications that currently do not have a spy panel, for example Timelion and X-Pack Monitoring.
```

Comment:
s/logQuerie/logQueries

Reply:
Maybe a slight simplification for the first sentence?  
> Logs queries sent to Elasticsearch.  Requires `logging.verbose` set to true.

`Kibana Elasticsearch server client` tripped me up when I read it the first time